interface-definitions/container.xml.in

```diff
@@ -192,6 +192,24 @@
               </leafNode>
             </children>
           </tagNode>
+          <leafNode name="cpus">
+            <properties>
+              <help>CPUS available to this container</help>
+              <valueHelp>
+                <format>u32:0</format>
+                <description>Unlimited</description>
+              </valueHelp>
+              <valueHelp>
+                <format>txt</format>
+                <description>Container cpus in cores (up to three decimals)</description>
+              </valueHelp>
+              <constraint>
+                <regex>^(0|[1-9]\d*)(\.\d{1,3})?$</regex>
```

Comment:
`^` and `$` are not needed. They will be added automatically to the resulting regex

Reply:
fixed